pkg/field/defaults.go

```
@@ -45,9 +62,38 @@
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelCollectorEndpointTlsCertPath = StringField(OtelCollectorEndpointTLSCertPathFieldName, WithDescription("Path to a file containing a PEM-encoded certificate to use as a CA for TLS connections to the OpenTelemetry collector"),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelCollectorEndpointTlsCert = StringField(OtelCollectorEndpointTLSCertFieldName, WithDescription("A PEM-encoded certificate to use as a CA for TLS connections to the OpenTelemetry collector"),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelCollectorEndpointTlsInsecure = BoolField(OtelCollectorEndpointTLSInsecureFieldName, WithDescription("Allow insecure connections to the OpenTelemetry collector"),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelTracingDisabled = BoolField(OtelTracingDisabledFieldName, WithDescription("Disable OpenTelemetry tracing"), WithDefaultValue(false),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelLoggingDisabled = BoolField(OtelLoggingDisabledFieldName, WithDescription("Disable OpenTelemetry logging"), WithDefaultValue(false),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelTracingEndpoint = StringField(OtelTracingEndpointFieldName, WithDescription("The endpoint of the OpenTelemetry collector to send tracing data to (overrides otel-collector-endpoint for tracing)"),
```

golangci-lint (1.64.8)

[error] line-length-limit: line is 203 characters, out of limit 200

(revive)